services/udp/service.go

```diff
@@ -0,0 +1,36 @@
+package udp
+
+import (
+	"net"
+	"strconv"
+	"time"
+)
+
+type Service struct {
+	Server *Server
+
+	iface string
+}
+
+func NewService(c Config) *Service {
+	server := NewServer(c.Database)
+	server.SetBatchSize(c.BatchSize)
+	server.SetBatchTimeout(time.Duration(c.BatchTimeout))
+
+	return &Service{
+		iface:  net.JoinHostPort(c.BindAdress, strconv.Itoa(c.Port)),
```

Comment:
We are moving to a `addr` variable name here.  Also, we no longer use port as part of the config.  `BindAddress` from the config should now be in the format of `host:port`.